src/Common/src/TypeSystem/Interop/IL/NativeWrapperDelegate.Sorting.cs

```diff
@@ -0,0 +1,17 @@
+// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+// See the LICENSE file in the project root for more information.
+
+namespace Internal.TypeSystem.Interop
+{
+    // Functionality related to determinstic ordering of types
+    partial class NativeWrapperDelegate
```

Comment:
This is confusing. This is not really a delegate, but a wrapper that wraps native function pointer, and you'll have a delegate point to it.